tests/exec/mod.rs

```diff
@@ -75,7 +75,7 @@ fn unix_echo() {
     handle.join().unwrap();
 }
 
-#[cfg(feature = "backend-sev")]
+#[cfg(use_shim)]
```

Comment:
this should already be applied in the `integration.rs`

Reply:
removed